.github/workflows/linux.yml

```diff
@@ -1,4 +1,4 @@
-name: Linux (Ubuntu 20.04, Python 3.11)
+name: Linux (Ubuntu 22.04, Python 3.11)
```

Comment:
Could you please recall why we have decided to use 22 ubuntu?

Reply:
Ubuntu 20 has the default Python 3.8, and we're dropping it 's support